MANIFEST.in

```diff
@@ -9,6 +9,7 @@ include src/rhub/bare_metal/model/data/kickstart_debug_script.sh
 include src/rhub/bare_metal/tasks/playbooks/*.yml
 
 recursive-include migrations *.py
-recursive-include config *yaml
+recursive-include config *.yaml
+recursive-include config *.json
```

Comment:
Some of the files ignored on `config/.gitignore` may be part of the package while testing locally, but not part of it when tested on CI. This may lead to undefined behavior.

I suggest to add the content of `config/.gitignore` on `check-manifest` ignore list or to itemize the files you want on the final package.

Reply:
Working on that in a way that's more specific. For me - as it is - adding to config/.gitignore didn't work (just discovered why but it's a different story altogether.) I chose to itemize things for now.

Reply:
This was also solved via itemizing.